src/calculation/ClauseResultsBuilder.ts

```diff
@@ -363,100 +372,110 @@ export function prettyResult(result: any | null, indentLevel?: number, keyIndent
   if (result instanceof DateTime) {
     return moment.utc(result.toString()).format('MM/DD/YYYY h:mm A');
   } else if (result instanceof Interval) {
-    return `INTERVAL: ${prettyResult(result.low)} - ${prettyResult(result.high)}`;
+    const typeStr = includeType ? 'INTERVAL: ' : '';
+    return `${typeStr}${prettyResult(result.low)} - ${prettyResult(result.high)}`;
   } else if (result instanceof Code) {
-    // TODO: Sort out a better way to have a friendly system display for FHIR codes
-    return `CODE: ${result.system} ${result.code}`;
+    const typeStr = includeType ? 'CODE: ' : '';
+    if (result.system) {
+      if (systemMap[result.system]) {
```

Comment:
Slightly more idiomatic to check if the key exists in the object rather than checking if its value is truthy

```suggestion
      if (result.system in systemMap) {
```